services/admin/src/util/store.ts

```diff
@@ -0,0 +1,28 @@
+/* istanbul ignore file */
+import { Store } from '../store';
+
+/**
+ * Adds a CVR file to the store with default test data
+ */
+export function addTestCvrFile(store: Store): string {
```

Comment:
Since this file only contains test utils, might make sense to name the file / directory accordingly, e.g. `src/test-util/`